modules/profile/manifests/st2rbac.pp

```diff
@@ -5,6 +5,7 @@
   $_enterprise_token = hiera('st2enterprise::token', undef)
 
   $_root_cli_username = $::profile::st2server::_root_cli_username
+  $_chatops_bot_username = 'chatops_bot'
```

Comment:
Cool, thanks, will change it to use this.

On a related note - we don't create system (linux) user for chatops_bot, right?